app/src/test/java/gateway/ITServiceAuth.java

```diff
@@ -53,4 +54,25 @@ class ITServiceAuth
 			500, ServiceAuth.getUserUUID (res.auth.token, cred.username).code,
 			"Can't reach Auth Server");
 	}
+
+	@Test void Login ()
+	{
+		Credentials cred = new Credentials (UUID.randomUUID ().toString (), "pass");
+		// register
+
+		ResSession res = CtrlAccountRegister.account_register (cred);
+		String token = res.auth.token;
+		assertEquals (201, res.code, "Login successfully");
+
+		ResSession login = CtrlAuthLogin.auth_login (cred);
+		assertEquals (200, login.code, "Login succeed");
```

Comment:
I see you found we have `200` in the spec but the auth service returns `201` :open_mouth: 

In that case, let's use only `201` for now on to avoid confusion.

Please update the Open API Specification accordingly.